Flaky test: test_max_rows_to_read_leaf_with_view/test.py::test_max_rows_to_read_leaf_via_view

_Important: This issue was automatically generated and is used by CI for matching failures. DO NOT modify the body content. DO NOT remove labels._

Test name: test_max_rows_to_read_leaf_with_view/test.py::test_max_rows_to_read_leaf_via_view
Failure reason: AssertionError
CI report: [Integration tests (arm_binary, distributed plan, 2/4)](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=93915&sha=6b912d4db652975469e8c9a85986b4bcc72e76a0&name_0=PR&name_1=Integration%20tests%20%28arm_binary%2C%20distributed%20plan%2C%202%2F4%29)

Failing test history: [cidb](https://play.clickhouse.com#V0lUSAogICAgOTAgQVMgaW50ZXJ2YWxfZGF5cwpTRUxFQ1QKICAgIHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBBUyBkYXksCiAgICBjb3VudCgpIEFTIGZhaWx1cmVzLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpIEFTIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgKG5vdygpIC0gdG9JbnRlcnZhbERheShpbnRlcnZhbF9kYXlzKSkgPD0gY2hlY2tfc3RhcnRfdGltZQogICAgQU5EIHRlc3RfbmFtZSA9ICd0ZXN0X21heF9yb3dzX3RvX3JlYWRfbGVhZl93aXRoX3ZpZXcvdGVzdC5weTo6dGVzdF9tYXhfcm93c190b19yZWFkX2xlYWZfdmlhX3ZpZXcnCiAgICAtLSBBTkQgY2hlY2tfbmFtZSA9ICdJbnRlZ3JhdGlvbiB0ZXN0cyAoYXJtX2JpbmFyeSwgZGlzdHJpYnV0ZWQgcGxhbiwgMi80KScKICAgIEFORCB0ZXN0X3N0YXR1cyBJTiAoJ0ZBSUwnLCAnRVJST1InKQogICAgQU5EIChwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCBPUiBiYXNlX3JlZiBJTiAoJ21hc3RlcicpKQogICAgQU5EIHRlc3RfY29udGV4dF9yYXcgTElLRSAnJUFzc2VydGlvbkVycm9yJScKR1JPVVAgQlkgZGF5Ck9SREVSIEJZIGRheSBERVNDCg==)

Test output:
```
File: test_max_rows_to_read_leaf_with_view/test.py:55 - in test_max_rows_to_read_leaf_via_view
    assert (
E   AssertionError: assert '200' == '400'
E     
E     - 400
E     + 200
```

--- a/tests/integration/test_max_rows_to_read_leaf_with_view/test.py
+++ b/tests/integration/test_max_rows_to_read_leaf_with_view/test.py
@@ -75,9 +75,9 @@ def test_max_rows_to_read_leaf_via_view(started_cluster):
         )
 
     for node in (node1, node2):
-        node.query("DROP VIEW IF EXISTS test_view")
-        node.query("DROP TABLE IF EXISTS distributed_table")
-        node.query("DROP TABLE IF EXISTS local_table")
+        node.query("DROP VIEW IF EXISTS test_view SYNC")
+        node.query("DROP TABLE IF EXISTS distributed_table SYNC")
+        node.query("DROP TABLE IF EXISTS local_table SYNC")
 
 
 if __name__ == "__main__":

--- a/tests/integration/test_max_rows_to_read_leaf_with_view/test.py
+++ b/tests/integration/test_max_rows_to_read_leaf_with_view/test.py
@@ -35,7 +35,7 @@ def test_max_rows_to_read_leaf_via_view(started_cluster):
     for node in (node1, node2):
         node.query(
             f"""
-            CREATE TABLE local_table(id UInt32, d DateTime) ENGINE = ReplicatedMergeTree('/clickhouse/tables/0/max_rows_read_leaf', '{node}') PARTITION BY toYYYYMM(d) ORDER BY d;
+            CREATE TABLE local_table(id UInt32, d DateTime) ENGINE = ReplicatedMergeTree('/clickhouse/tables/0/max_rows_read_leaf', '{node.name}') PARTITION BY toYYYYMM(d) ORDER BY d;
 
             CREATE TABLE distributed_table(id UInt32, d DateTime) ENGINE = Distributed(two_shards, default, local_table);
 
@@ -45,6 +45,10 @@ def test_max_rows_to_read_leaf_via_view(started_cluster):
 
     node1.query(
         "INSERT INTO local_table (id) select * from system.numbers limit 200"
+    )
+
+    node2.query(
+        "SYSTEM SYNC REPLICA local_table"
     )
 
     """

